homeassistant/components/greeneye_monitor/sensor.py

```diff
@@ -239,7 +240,7 @@ def _seconds_per_time_unit(self) -> int:
             return 3600
 
         # Config schema should have ensured it is one of the above values
-        raise Exception(
+        raise HomeAssistantError(
```

Comment:
This sounds like a case that should not be able to happen. I'd raise RuntimeError for now until a refactor can remove this case.